modules/subchannel/validation/Toshiba_37_pin/gold/toshiba_37_pin_out.csv

```diff
@@ -1,3 +1,3 @@
 time,DP_SubchannelDelta,T_Planar_Mean
 0,0,0
-1,9286.5384435489,707.4469412441
+1,9286.6619132485,707.44050544809
```

Comment:
Small enough changes. No appreciable difference within experimental uncertainties of the test.